tests/python/unittest/test_gluon_data.py

```diff
@@ -285,6 +285,11 @@ def test_multi_worker_dataloader_release_pool():
 
 
 def test_dataloader_context():
+    if os.name == 'nt':
+        print("Skipping test_dataloader_context on Windows due to "
+              "https://github.com/apache/incubator-mxnet/issues/17961")
```

Comment:
Please change back to G3.

Reply:
Please provide a technical reason for using g3. Currently Jenkins faces connectivity issues on g3 instance.

Reply:
G3 represent the minimum we are verifying. G4 is compatible with G3 but not the other way around. 

Do not change things if they do not work. I want to see some quality check before changes like this are made without checking with the community.

Reply:
We are still verifying g3 on unix gpu tests. There is no difference in the cuda kernels between windows and unix. It's sufficient to have one pipeline run on g3 to verify the compatibility.